kpi-tracker/src/main.rs

```diff
@@ -27,14 +27,17 @@ struct Args {
         help = "The endpoint is expected to point to a concordium node grpc v2 API.",
         default_value = "http://localhost:20001"
     )]
-    node:       Endpoint,
+    node:        Endpoint,
     /// How many blocks to process.
     // Only here for testing purposes...
     #[arg(long = "num-blocks", default_value_t = 10000)]
-    num_blocks: u64,
+    num_blocks:  u64,
     /// Logging level of the application
     #[arg(long = "log-level", default_value = "debug")]
-    log_level:  log::LevelFilter,
+    log_level:   log::LevelFilter,
+    /// Block height to start collecting from
+    #[arg(long = "from-height", default_value_t = 0)]
```

Comment:
Why is `_t` needed? Is that some new thing? It used to be just `default_value`.

Reply:
`_t` can be used to specify a default value in the type of the field. It has to implement `std::format::Display`, which is why it hasn't been used consistently for all fields.

Reply:
One important point is that the Display implementation must then be compatible with FromStr. Otherwise the help text will be confusing. Is that the case here?

Reply:
True, didn't think of that tbh but in the case where I use it they are compatible.